src/test/ui/no-landing-pads.rs

```diff
@@ -1,6 +1,7 @@
 // run-pass
 // compile-flags: -Z no-landing-pads -C codegen-units=1
 // ignore-emscripten no threads support
+// ignore-test fails because catch_unwind doesn't work with no-landing-pads
```

Comment:
I think it's fine to just delete this test. This is testing the `-Z` option but it's not too important to test that nowadays. This could be re-introduced by having separate compilation, one crate with the flag and one without.